agent.env

```diff
@@ -0,0 +1,169 @@
+
+CONNECT_TIMEOUT=10
+MAX_CONNECTIONS=1000
+IDLE_TIMEOUT=30000
+SESSION_ACQUIRE_TIMEOUT=2147483647
+SESSION_LIMIT=2147483647
+INMEMORY_LRU_CACHE_LIMIT=2147483647
+BCOVRIN_REGISTER_URL=http://test.bcovrin.vonx.io/register
+INDICIO_NYM_URL=https://selfserve.indiciotech.io/nym
+SCHEMA_MANAGER_CONTRACT_ADDRESS=0x289c7Bd4C7d38cC54bff370d6f9f01b74Df51b11
+FILE_SERVER_TOKEN=
+RPC_URL=https://rpc-amoy.polygon.technology
```

Comment:
We should use the mainnet configuration polygon as default value.

Reply:
We are using this script for provisioning the agent locally. I think we don't need mainnet configurations for local use. What do you think?

Reply:
Resolved